changelog/_unreleased/2024-12-03-reorganized-menu-items-on-the-settings-page.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Reorganized menu items on the settings page
+issue: NEXT-37079
+flag: v6.7.0.0
+author: Sebastian Kalicki
+author_email: s.kalicki@shopware.com
+author_github: @s.kalicki
+---
+# Administration
+* Changed `settings-item.store.js` to extend the state for supporting the reorganized menu structure on the settings page.
+* Changed `sw-settings-index.html.twig` to implement the new groups and reflect the updated menu structure.
+
+___
+
+# Upgrade Information
+## Updated Menu Structure in Settings Page
+The settings page has been reorganized into groups for better usability. If you extend or customize the settings menu, ensure that your changes are compatible with the new structure.
+
+### Details:
+* Changed settings-item.store.js to extend the state for supporting the reorganized menu structure on the settings page.
+* Changed sw-settings-index.html.twig to implement the new groups and reflect the updated menu structure.
+
+### Code Updates:
```

Comment:
This is a little to detailed ✌️ 🙈. Just think from a perspective of an external plugin developer. For me the most important change is:
- The new available groups and that I should try to choose one of these, or if they don't fit use my own
- The way I can register my code for that group (in the module register call and that it can be a function with logic or just a string)
- did we remove or change some twig blocks or app extension points that I might relied on?